utils.js

```diff
@@ -1,11 +1,13 @@
 // baseline estimates, used to improve performance
 var TX_EMPTY_SIZE = 4 + 1 + 1 + 4
 var TX_INPUT_BASE = 32 + 4 + 1 + 4
+var TX_INPUT_SEGWIT = 34
```

Comment:
@Turtus Why 34?

btw, a transaction input will always have at least 41 bytes (`TX_INPUT_BASE = 32 + 4 + 1 + 4`).

- 32 bytes for the [previous txid hash](https://learnmeabitcoin.com/technical/txid)
- 4 bytes for the ([vout](https://learnmeabitcoin.com/technical/vout))
- 1 byte for the scriptSigSize [varint](https://learnmeabitcoin.com/technical/varint)
- 4 bytes for the sequence

https://learnmeabitcoin.com/technical/input